lib/chai/core/assertions.js

```diff
@@ -537,10 +537,11 @@ module.exports = function (chai, _) {
         itemsCount = val.length;
         break;
       case 'function':
-        throw new TypeError('.empty() was passed a function');
+        var name = val.name ? ' ' + val.name : '';
+        throw new TypeError('.empty was passed a function' + name);
       default:
         if (val !== Object(val)) {
-          throw new TypeError('.empty() was passed non-string primitive');
+          throw new TypeError('.empty was passed non-string primitive ' + String(val));
```

Comment:
LOL, THE LUCAS SYNDICATE IS WILD TODAY
So, what do we want here, `type` or `value`?